virttest/qemu_vm.py

```diff
@@ -662,6 +663,14 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                     dev.set_param('mq', 'on')
                 if vectors:
                     dev.set_param('vectors', vectors)
+                check_opt = {"indirect_desc": indirect_desc,
+                             "mrg_rxbuf": mrg_rxbuf,
+                             "ctrl_vq": ctrl_vq,
+                             "ctrl_vlan": ctrl_vlan,
+                             "status": status}
+                for opt in check_opt:
+                    if check_opt[opt] and check_opt[opt] in ["on", "off"]:
```

Comment:
@sitoliu  Seems this line equal to 'if check_opt[opt] in ["on", "off"]:', right?